sources/ca/on/city_of_kingston.json

```diff
@@ -18,24 +18,31 @@
                 "name": "city",
                 "protocol": "http",
                 "compression": "zip",
-                "data": "https://apps.cityofkingston.ca/data/gis/Civic_Address_SHP.zip",
-                "website": "https://www.cityofkingston.ca/explore/data-catalogue",
+                "data": "https://opendatakingston.cityofkingston.ca/explore/dataset/civic-addresses/download/?format=geojson&timezone=America/New_York&lang=en",
```

Comment:
Looks like you're asking to download it as geojson but specify a format of "shapefile" below.